docs/source/contributing/documentation.rst

```diff
@@ -309,7 +309,7 @@ dictionary that maps a string to either an int or an instance of
    a list, ``Tuple[type_a, type_b, (...), type_n]`` for a tuple (if the
    elements are all different) or ``Tuple[type, ...]`` (if all elements
    have the same type). Each ``type_n`` on those representations
-   correspond to elements in the returned list/tuple, and must follow
+   corresponds to elements in the returned list/tuple and must follow
```

Comment:
Pretty sure the original version is correct.

Reply:
This is grammatically more correct so I think we should have it

Reply:
You're right